src/bindeps_conda.jl

```diff
@@ -1,57 +1,46 @@
 # This file contains the necessary ingredients to create a PackageManager for BinDeps
 using BinDeps
 
-type Manager <: BinDeps.PackageManager
+type ManagerType{T} <: BinDeps.PackageManager
     packages::Vector{Compat.ASCIIString}
 end
 
-function Base.show(io::IO, manager::Manager)
+"Manager for root environment"
+Manager = ManagerType{Symbol(PREFIX)}
+
+function Base.show{T}(io::IO, manager::ManagerType{T})
     write(io, "Conda packages: ", join(manager.packages, ", "))
 end
 
-BinDeps.can_use(::Type{Manager}) = true
+BinDeps.can_use{T}(::Type{ManagerType{T}}) = true
```

Comment:
The type parameter is not necessary since you don't use it.  Just `BinDeps.can_use(::Type{ManagerType}) = true` suffices.